src/utils/Logger.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Define a custom Logger using winston
+ */
+
+import * as winston from "winston";
+
+export const logger = new winston.Logger({
+  level: "debug",
```

Comment:
level should come from an env var, we don't want to have production code logging in debug mode, do we?